### DISCUSSION TITLE
I'm doing an internship

### DISCUSSION THREAD

**Discussion by Durmpoe** [community/community#113010] [GitHub Education]
### Hi everyone,

Hi,

I am currently doing an internship abroad. Someone I work with mentioned GitHub Education, and I am interested in applying for the benefits.

My school does not issue student ID cards with dates etc., but I can prove my enrollment by showing my details. I have also received a school email with a "Verify email address" link.

Why is it so difficult to verify my student status? What can I do to ensure that I am verified?

Thank you for your time

**Comment by SebastienXia** [comment]
## Hello, @Durmpoe 👋

📜 Registration is not as difficult as it seems, you just have to follow the process.

And to prove that you are registered in an educational establishment you have the possibility of proving it with different proofs, for example your registration for the 2023-2024 school year.

> Furthermore, you can use your personal email for registration.

![310121077-7599c586-817d-4fd3-955d-5133b897670b](https://github.com/community/community/assets/105004912/1715335e-40ac-4f3b-a359-640a9cb0c97b)


`[close the thread and mark as answered if this helped you resolve your issue]`

**Reply by Durmpoe** [reply]
If I follow everything I still get that and what I already am I am not close to school I am abroad
![Screenshot 2024-03-17 194500](https://github.com/community/community/assets/116366385/a8233f2b-189e-470f-afcb-fda70e48f756)

**Comment by samus-aran** [comment]
Hi @Durmpoe 👋 

Your evidence needs to be able to prove that you are currently enrolled at your school with correspondence addressed to you. Important information on your evidence should identify yourself and your school.

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

**Reply by Durmpoe** [reply]
Thanks for your message. It's difficult that you can't submit screenshots anywhere. It is also only possible to show something with the camera (if that option is there, there is no idea where that option is.) I don't understand why things are being done so strictly. I have a school email address that I use and I just send an email to it and you have proof that I go to that school, that's all it really is. and how GitHub could control it is to check each other if it exists, that email, yes, then you will receive the education. If the email does not exist, you are not entitled to it. I think I'll just leave it alone and wait until my internship is over and try again on location.

Thanks again for your message
